app/src/androidTest/java/com/reach5/identity/sdk/demo/SandboxSuccessTest.kt

```diff
@@ -0,0 +1,214 @@
+package com.reach5.identity.sdk.demo
+
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.allOf
+import org.hamcrest.TypeSafeMatcher
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.*
+import java.util.regex.Pattern
+
+@LargeTest
+@RunWith(AndroidJUnit4::class)
+class SandboxSuccessTest {
```

Comment:
Indentation does not seem like the rest of the project.

Reply:
Done